docs/docs/02200-tables/00100-overview.md

```diff
@@ -0,0 +1,128 @@
+---
+title: Overview
+slug: /tables
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Tables
+
+Tables are the primary way to store data in SpacetimeDB. All data in SpacetimeDB is stored in memory for extremely low latency and high throughput access. SpacetimeDB also automatically persists all data to disk.
```

Comment:
"primary" feels odd here, as it implies there are other (secondary) ways to store data, which there aren't.